### ISSUE TITLE
{Linked:Bug9780999;Linked:Bug9768636;Freeway-Bug} [Adaptive Cards> Designer]: Unable to access 'year' links using tab key that appears after clicking on 'month, year' dropdown in date picker.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8098]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25247.1000)
Browser: Edge Dev (109.0.1495.2)

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Navigate to the 'Designer' tab through tab key and activate it.
3. Navigate to the 'Sample data Editor' and ' Card payload editor' and attach the provided JSON file.
4. Navigate to 'Preview mode' and activate it.
5. Navigate to 'Enter a Date' button and activate it.
6. Navigate to the 'Year' option and press enter key to activate it.
7. Press tab key and observe the issue.

**Actual Result:**
'Year' links are not accessible using tab key that appears after clicking on 'month, year' dropdown in date picker.

**Expected Result:**
'Year' links should be accessible using tab key that appears after clicking on 'month, year' dropdown in date picker.
 
**User Impact:**
It will impact the keyboard user if the year link is not accessible via keyboard as keyboard dependent user might be miss to use the control.

**MAS Reference:**
[MAS 2.1.1 – Keyboard
](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.01)

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/205305421-72bea5dc-dc6f-4f4c-af6a-c89a5a5503a0.mp4

### Card JSON

```JSON
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.3",
  "body": [
    {
      "type": "TextBlock",
      "text": "Date Input"
    },
    {
      "type": "Input.Date",
      "id": "date",
      "placeholder": "Enter a date",
      "value": "2017-10-12"
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "OK"
    }
  ]
}
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

I am closing this issue because the date picker is keyboard accessible via the arrow keys. Below you can see that after navigating to the "month, year" dropdown, the arrow keys can be used to navigate between controls.

![dateKeyboardNav](https://user-images.githubusercontent.com/98650930/215615690-ba086053-7b78-49d1-9723-b666d63a1027.gif)

Let me know if you have any questions - thanks!

**Comment by Raisul123** [comment]
Hi @anna-dingler yes agree with your comment as Different years are accessible via the arrow keys. But it will impact keyboard user as they have to put extra effort to navigate Different years.

**Comment by anna-dingler** [comment]
Hi @Raisul123 and @vagpt,

My team has no control over the contents of the HTML input date control, so we are unable to modify the keyboard navigation.

I tested the following HTML on both Edge and Chrome, and the behavior is the same as on our designer (tab key does not work, arrow keys work).

**Test HTML**
```HTML
<input type="date">
```

Since this behavior is out of our control, would it be possible to close this bug?

**Comment by anna-dingler** [comment]
Closing per the above comment.

**Comment by vagpt** [comment]
Closing this as per the above comment and adding A11yTTValidated tag.
